app/forms/idv/finance_form.rb

```diff
@@ -5,7 +5,10 @@ class FinanceForm
     FINANCE_TYPES = [:ccn, :mortgage, :home_equity_line, :auto_loan].freeze
 
     FINANCE_HTML_OPTIONS = {
-      ccn: { maxlength: 8 }
+      ccn: { class: 'ccn', pattern: '[0-9]*', maxlength: 8 },
```

Comment:
if you rebase off of master, this constant should include some other constants :)